node/src/RedisClusterClient.ts

```diff
@@ -56,7 +57,7 @@ export type SlotKeyTypes = {
 export type RouteByAddress = {
     type: "routeByAddress";
     /**
-     * DNS name of the host.
+     * The name of the node. If `port` is not provided, should be in the `{DNS/ip}:{port}` format, as returned from the cluster when sending a request to multiple nodes.
```

Comment:
I don't think it's a good way of explaining it, as the results of some commands that are being sent to multiple nodes are being aggregated to a single response.. so the user really needs to know the client to understand that. Can you check if we can use cluster nodes/slots output to explain what should be passed?